bin/search.py

```diff
@@ -1,8 +1,9 @@
 import subprocess
 import sys
+import json
 import logging
-import louis.db.api as api
-import louis.db as db
+import ailab.db.api as api
+import ailab.db as db
 from microbench import MicroBench, MBReturnValue, MBFunctionCall
 import pandas as pd
```

Comment:
The lint fail point to this line with message: "bin/search.py:8:18: F401 `pandas` imported but unused". You can also check the reason for failure in the repo's "action" tab: https://github.com/ai-cfia/ailab-db/actions/runs/6945568744/job/18895234952?pr=50

Reply:
forgot about that, done